charts/kyverno-policies/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+name: kyverno-policy
+version: v2.3.0
+appVersion: v1.6.0
+icon: https://github.com/kyverno/kyverno/raw/main/img/logo.png
+description: Kubernetes Pod Security Standards implemented as Kyverno policies
+keywords:
+  - kubernetes
+  - nirmata
+  - policy agent
+  - validating webhook
+  - admissions controller
+home: https://kyverno.io/policies/
```

Comment:
This is not related to the upstream Kyverno policies repo. We shouldn't reference it here.

Reply:
Okay